File: Codeforces/div4/928/C_Vlad_and_a_Sum_of_Sum_of_Digits.java
import java.util.Scanner;

public class C_Vlad_and_a_Sum_of_Sum_of_Digits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int t = scanner.nextInt(); // Number of test cases

        for (int testCase = 0; testCase < t; testCase++) {
            int n = scanner.nextInt(); // Largest number Vladislav writes

            // Calculate the number of complete cycles of 1 to 9
            int completeCycles = (n - 1) / 9;

            // Calculate the remaining digits after the last complete cycle
            int remainingDigits = n - 9 * completeCycles;

            // Calculate the sum of digits for the complete cycles
            int sumCompleteCycles = 45 * completeCycles;

            // Calculate the sum of digits for the remaining numbers
            int sumRemainingDigits = 0;
            for (int i = 1; i <= remainingDigits; i++) {
                sumRemainingDigits += i % 10;
            }

            // Calculate the total sum
            int totalSum = sumCompleteCycles + sumRemainingDigits;

            System.out.println(totalSum);
        }

        scanner.close();
    }
}

File: Codeforces/div4/928/C_Vlad_and_a_Sum_of_Sum_of_Digits.py
def sum_of_digits(number):
    return sum(int(digit) for digit in str(number))


def calculate_total_sum(n):
    total_sum = sum(sum_of_digits(i) for i in range(1, n + 1))
    return total_sum


def main():
    t = int(input())
    for _ in range(t):
        n = int(input())
        result = calculate_total_sum(n)
        print(result)


if __name__ == "__main__":
    main()
